packages/editor-ui/src/components/AskAssistant/AskAssistantChat.vue

```diff
@@ -27,14 +27,16 @@ function onResizeDebounced(data: { direction: string; x: number; width: number }
 }
 
 async function onUserMessage(content: string, quickReplyType?: string, isFeedback = false) {
-	await assistantStore.sendMessage({ text: content, quickReplyType });
-	const task = 'error';
-	const solutionCount =
-		task === 'error'
-			? assistantStore.chatMessages.filter(
-					(msg) => msg.role === 'assistant' && !['text', 'event'].includes(msg.type),
-				).length
-			: null;
+	// If there is no current session running, initialize the support chat session
+	if (!assistantStore.currentSessionId) {
+		await assistantStore.initSupportChat(content);
+	} else {
+		await assistantStore.sendMessage({ text: content, quickReplyType });
+	}
+	const task = assistantStore.isSupportChatSessionInProgress ? 'support' : 'error';
+	const solutionCount = assistantStore.chatMessages.filter(
```

Comment:
I know it's not related to this PR. But the backend should be returning this. We discussed this in planning but dropped in handover and final implementation. 
I guess it's fine since we are likely to drop this feature any way, so no need to change anything.